packages/node_modules/pouchdb-core/src/adapter.js

```diff
@@ -588,10 +579,18 @@ AbstractPouchDB.prototype.get = adapterFun('get', function (id, opts, cb) {
 
     if (opts.revs || opts.revs_info) {
       var paths = rootToLeaf(metadata.rev_tree);
-      var path = arrayFirst(paths, function (arr) {
-        return arr.ids.map(function (x) { return x.id; })
-          .indexOf(doc._rev.split('-')[1]) !== -1;
-      });
+      var path = paths.reduce(function selectRevPath(result, arr) {
+        var splittedRev = doc._rev.split('-');
+        var revNo       = parseInt(splittedRev[0]);
+        var revHash     = splittedRev[1];
+        var hashIndex   = arr.ids.map(function (x) { return x.id; })
+          .indexOf(revHash);
```

Comment:
No problem. That is because I tried to follow the way code was written before with arrayFirst. Unless you want to do it yourself, I will send you the updates in another PR.